### PR TITLE
Fix #86. Add Function Apps into Azure Explorer

### LINKED ISSUE #86: Functions: list function apps in Azure explorer
Similar to web apps

### PR DESCRIPTION
**Pull request by sdubov**
- Add Function Apps into Azure Explorer
- Reformat Azure Explorer and combine Web Apps and Function Apps into App Services node
- Add icons for Azure Function Apps
- Add Mvp class for operations with Azure Function Apps

### CODE REVIEW

File: Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/functionapp/AzureFunctionAppMvpModel.kt [hunk 1]
```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (c) 2018 JetBrains s.r.o.
```
Comment (ForNeVeR) [inline Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/functionapp/AzureFunctionAppMvpModel.kt]: I think it's time to update the copyright year :)

**Comment by maartenba** [comment]
Gave it a test run, looks good to me!

Random side note: would be nice to be able to list functions in a function app like in the portal:

image

There is an API for that (https://docs.microsoft.com/en-us/rest/api/appservice/webapps/listfunctions) but seems that is not in the core libraries we currently use.

**Review by maartenba — APPROVED** [review]

File: Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/functionapp/AzureFunctionAppMvpModel.kt (hunk 1 above)
Comment (sdubov) [reply Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/functionapp/AzureFunctionAppMvpModel.kt]: Sure :) Updated year for all new files. I'll make a separate commit with already existing files.

**Comment by sdubov** [comment]
@maartenba, good point. Created #159.

**Pull request merged**